src/imitation_loss/imitation_loss.jl

```diff
@@ -0,0 +1,52 @@
+"""
+    ImitationLoss{L,R,P}
+
+Generic imitation loss: max_y ℓ(y, y̅) + θᵀ(y - y̅) - (Ω(y) - Ω(y̅)).
+
+When `ℓ = 0`, this loss is equivalent to a [`FenchelYoungLoss`](@ref).
+When `Ω = 0`, this loss is equivalent to the [`StructuredSVMLoss`](@ref).
+
+# Fields
+- `base_loss::L`: base loss `ℓ`
```

Comment:
Is it coherent to spell it out here but use the unicode for Omega below? It doesn't bother me personally but consistency is nice too